### DISCUSSION TITLE
购买了2年的专业设备，但登录不了无法使用

### DISCUSSION THREAD

**Discussion by xyfreeloom** [getlantern/lantern-forum-cn#776] [Q&A]
购买了2年的专业设备，但登录不了无法使用，邮箱447979898@qq.com
![Uploading 20240812214555.png…]()

**Comment by MaxMKMax** [comment]
@xyfreeloom 您好. 您方便发您的支付单据到我的邮箱吗? 我这里看不到您的支付单据. 被Github屏蔽了.
mike@getlantern.org

我会帮您提交问题, 过 2 天请您查看这个帖子个帖子还是邮箱. 谢谢您支持蓝灯!

**Reply by xyfreeloom** [reply]
邮件也发了，还是不能用

**Reply by MaxMKMax** [reply]
@xyfreeloom 请问您是收不到激活码还是验证码对吗?

**Reply by xyfreeloom** [reply]
是的，没有收到激活码，想通过邮箱进行设备认证也收不到验证码

**Reply by MaxMKMax** [reply]
@xyfreeloom 好的. 我幫您提交問題上去.

**Comment by MaxMKMax** [comment]
@xyfreeloom 
您好. 这是您的验证码:
C7XRT-8X23K-DGR9P-VR9GQ-2MCWM
不好意思让您久等了.

**Comment by xyfreeloom** [comment]
好的，多谢




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年8月28日(星期三) 晚上11:26
***@***.***&gt;;
抄送:&nbsp;" ***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 购买了2年的专业设备，但登录不了无法使用 (Discussion #776)





 
@xyfreeloom
 您好. 这是您的验证码:
 C7XRT-8X23K-DGR9P-VR9GQ-2MCWM
 不好意思让您久等了.
